examples/lang/bad/unificationerror.mcl

```diff
@@ -0,0 +1,2 @@
+# this file generates a unification error, used in lib/cli_test.go
```

Comment:
okay, however why not write this as a test in unification_test.go instead?

Reply:
i want to test validate is able to detect a unification error, not that this file generates a unification error per se.